From b48bbc33830a4bef183cd6ec1119a08e54bc2c5e Mon Sep 17 00:00:00 2001
From: Meticulus <theonejohnnyd@gmail.com>
Date: Thu, 30 Nov 2017 15:30:19 -0600
Subject: [PATCH] "0001-zygote-white-list-for-aosp-boot-critial
 _______________________________

*Previous From ec013e3203e2cdd4d57f6d216fc34f06c9949bc8 Mon Sep 17 00:00:00 2001
*Previous From: Meticulus <theonejohnnyd@gmail.com>
*Previous Date: Wed, 25 Oct 2017 14:16:13 -0500
*Previous Subject: [PATCH] "0001-zygote-white-list-for-aosp-boot-critial
*_______________________________
*
**Previous From 1adcba33d95223edc538e8c29d1e8db7a0cf3cd4 Mon Sep 17 00:00:00 2001
**Previous From: Meticulus <theonejohnnyd@gmail.com>
**Previous Date: Sat, 13 May 2017 09:48:20 -0500
**Previous Subject: [PATCH] 0001-zygote-white-list-for-slim-boot-critial
**
**Change-Id: I37f4cdb20b12481a708ad0e33c00f3dd63949e5d"
*
*Change-Id: I97d2189920648726d756ae76e9cbba917d627830"

Change-Id: I99411af9761dcb41502756e508ee44b5be048922
---
 core/jni/fd_utils-inl.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/core/jni/fd_utils-inl.h b/core/jni/fd_utils-inl.h
index e1ed541..4e8140d 100644
--- a/core/jni/fd_utils-inl.h
+++ b/core/jni/fd_utils-inl.h
@@ -56,6 +56,11 @@ static const char* kPathWhitelist[] = {
   "/dev/urandom",
   "/dev/ion",
   "/dev/dri/renderD129", // Fixes b/31172436
+  "/sys/firmware/devicetree/base/hisi,product_name",
+  "/sys/firmware/devicetree/base/hisi,modem_id",
+  "/hwvendor/phone.prop",
+  "/vendor/phone.prop",
+  "/dev/__properties__"
 };
 
 static const char* kFdPath = "/proc/self/fd";
-- 
2.7.4

